.travis.yml

```diff
@@ -58,7 +58,7 @@ after_success:
 # travis-ci will quit using the cache if an enviroment variable changes
 env:
   global:
-    - CACHE_VERSION=netcoreapp.2.1-sdk.2.1.300-rc1-008662
+    - CACHE_VERSION=netcoreapp.2.1-sdk.2.1.300-rc1
```

Comment:
Should this be `rc1-008673` like in global.json or `-final` as in `Weblistner.csproj`?

Reply:
@TravisEz13 I see you added `CACHE_VERSION` to this yml.  Searching travis-ci docs I don't see any mention of this var.  I assumed this was just an opaque string since travis-ci wouldn't know anything specific about our version of dotnet, so the build number shouldn't be necessary